src/chrome/content/rules/MoS.org.xml

```diff
@@ -0,0 +1,45 @@
+<!--
+	No working URL known:
+		triton.mos.org (403)
+		live-proxy.mos.org (403)
+		tessituralive.mos.org (403)
+		tessituraqa.mos.org (403)
+
+	Invalid certificate:
+		email.mos.org
+		etf.mos.org
+		mail.mos.org
+		mailarchive.mos.org
+		www.starwars.mos.org
+		team.mos.org
+		tracking.mos.org
+
+	Time out:
+		exhibits.mos.org
+		library.mos.org
+
+	Private subdomain:
+		kronos.mos.org
+		login.mos.org
+
+	Too many redirects:
+		legacy.mos.org
```

Comment:
I am not sure what you mean here. I check with `curl -v -L`and actually, all 3 of those subdomains redirect to `http`.